gabbi/tests/test_history.py

```diff
@@ -120,7 +121,10 @@ def test_cookie_replace_prior_regex(self):
 
         cookie = self.test('test_request').replace_template(
             self.test.test_data, escape_regex=True)
-        self.assertEqual(r'/test\=cookie\?/', cookie)
+        if sys.version_info[:2] >= (3, 7):
+            self.assertEqual('/test=cookie\?/', cookie)
+        else:
+            self.assertEqual('/test\=cookie\?/', cookie)
```

Comment:
You might wanna turn that bit from the commit message into a comment here - though I'm not actually convinced it's worth the bloat.

Reply:
Yeah, I waffled on that. This makes it pretty clear that _something_ changed in 3.7 but not what and I'd probably be annoyed by needing to git blame to find out what it was.

But then again, annoying people can be fun.